code/modules/hydroponics/seed_datums.dm

```diff
@@ -602,12 +648,103 @@
 	..()
 	set_trait(TRAIT_PRODUCT_COLOUR,"42ed2f")
 
+// Lettuce/varieties.
+/datum/seed/lettuce
+	name = "lettuce"
+	seed_name = "lettuce"
+	display_name = "lettuce"
+	chems = list("nutriment" = list(1,15))
+	kitchen_tag = "cabbage"
```

Comment:
*squint*

Reply:
This is to save the trouble of making 2 recipes for cabbage / lettuce salads.